docs/userGuide/syntax/popovers.mbdf

```diff
@@ -135,6 +137,22 @@ More about <trigger for="pop:trigger_id">trigger</trigger>.
 <popover id="pop:trigger_id" content="This popover is triggered by a trigger"></popover>
 <br>
 This is the same <trigger for="pop:trigger_id">trigger</trigger> as last one.
+<br>
+<popover content="You can select me!">This popover's content can be interacted with.</popover>
+</span>
+</include>
+
+**Hoverable popovers:**<br>
+
+<include src="outputBox.md" boilerplate >
+<span id="code">
+
+```html
+<popover content="You can select me!" hoverable>This popover's content can be interacted with.</popover>
+```
+</span>
+<span id="output">
+<popover content="You can select me!" hoverable>This popover's content can be interacted with.</popover>
```

Comment:
I'm wondering, why can't this be the default/only behavior? Is there any downside to making all popovers hoverable?

Reply:
I left a reason [here](https://github.com/MarkBind/vue-strap/pull/115#issuecomment-512628980). It is a small problem, but still tolerable enough to make it the only behavior.